package.json

```diff
@@ -57,8 +58,11 @@
     "eslint-plugin-import": "^2.17.3",
     "mocha": "^5.2.0",
     "nodemon": "^1.19.1",
-    "nyc": "^14.1.1",
-    "sinon": "^7.3.2"
+    "nyc": "^14.1.1"
+  },
+  "engines": {
+    "node": ">=6.0.0",
```

Comment:
why node `6` not `10` ?

Reply:
I wanted it to be backward compatible. That was why I used >= 6.